docs/demo/test-api.js

```diff
@@ -69,6 +70,18 @@ module.exports = function (config) {
       return Promise.resolve(ids[id])
     },
     tagsAndCategories: () => Promise.resolve(config.tagsAndCategories),
+    settings: () => Promise.resolve(config.settings),
+    setSetting: (name, value, addOptions) => {
```

Comment:
ahh yes, how about `addedOptions`. `addOptions` felt like it was going to be a boolean or something.